inc/tuoterivi.inc

```diff
@@ -271,15 +271,12 @@ if (mysql_field_name($result, $i) == "sarjanumeroseuranta") {
   if ($poistolukko == "" or $trow[$i] == "" or ($vrow["saldo"] == 0 and in_array($trow[$i], array("", "T", "V", "E", "F")))) {
     $ulos .= "<option value = ''>".t("Ei seurantaa")."</option>";
   }
-  if ($poistolukko == "" or $sel["S"] == "selected" or ($vrow["saldo"] == 0 and in_array($trow[$i], array("S", "U")))) {
+  if ($poistolukko == "" or $sel["S"] == "selected" or ($vrow["saldo"] == 0 and in_array($trow[$i], array("S")))) {
```

Comment:
@tiere Ehkä vähän gögä ku arrayssa on vaa yks elementti. 

    and in_array($trow[$i], array("S"))